packages/react/src/experimental/suspensiveCache.tsx

```diff
@@ -0,0 +1,113 @@
+import { Tuple } from '../types'
+
+type Cache<Key extends Tuple = Tuple> = {
+  promise?: Promise<unknown>
+  key: Key
+  error?: unknown
+  data?: unknown
+}
+
+class SuspensiveCacheObserver {
```

Comment:
Doesn't the forced reload(cmd + R) of Chrome blow the cache?
Even if I force a reload, the cached data remains.

Reply:
chrome will blow cache on reload. did you test it in Next.js 13? if right, I guess you meet already suspensed data in server without meeting Suspense's fallback

Reply:
Oh i understood
It works in server i think!